Async use of Link (#243)
codatio/codat-docs: docs/auth-flow/overview.md

We suggest using Link for best results, which can be [embedded within your app](/auth-flow/authorize-embedded-link). However, where you need full control of the flow, you can use our API to [build your own authorization journey](/auth-flow/build/build-your-own-authorization-journey).

## Link in practice

> 

Reviewer: Which of the three approaches does the guidance on async use of Link relate to? Hosted Link, embedded Link or Link API?

Author: can be used for all three, hence no specific callout

> Our auth flow solution supports many real-life applications and scenarios that you can levarage regardless of the auth flow approach you choose.

Reviewer: ```suggestion
Our auth flow solution supports many real world applications and scenarios that you can leverage regardless of the approach you take to building your auth flow.
```

> - **Asynchronous use of Link**

Reviewer: The single bullet list looks a bit odd in the preview.

> In your customer's organization, the person signing up with Codat may not have the credentials for the integration platform they want to connect. To enable them to proceed and explore the product, you can make the the authorization step optional.

Reviewer 2: ```suggestion
In your customer's organization, the person signing up with Codat may not have their credentials to hand. To enable them to proceed and explore the product, you can make upfront authorization optional. Later, remind them to authorize, providing a clear indication of the value to them. 
```

> In your customer's organization, the person signing up with Codat may not have their credentials to hand. To enable them to proceed and explore the product, you can make upfront authorization optional. Later, remind them to authorize, providing a clear indication of the value to them. 

Reviewer: Suggestions:

the person signing up with Codat > the user signing up through Codat

"explore the product" - which product does this refer to?

Reviewer: It could be clearer _how_ they can "make upfront authorization optional".

> Later, approach them with a request to authorize and provide them with an option to authorize themselves, or invite someone else. This can be done via email, or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.

Reviewer 2: ```suggestion
The user signing up may not have access to their business' financial data at all. For best results, provide them with an option to authorize themselves, or to invite someone else to (e.g. a member of their finance team). This can be done via email, or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.
```

> The user signing up may not have access to their business' financial data at all. For best results, provide them with an option to authorize themselves, or to invite someone else to (e.g. a member of their finance team). This can be done via email, or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.

Reviewer: business's financial data

> The user signing up may not have access to their business' financial data at all. For best results, provide them with an option to authorize themselves, or to invite someone else to (e.g. a member of their finance team). This can be done via email, or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.

Reviewer: ```suggestion
The user signing up may not have access to their business' financial data at all. For best results, provide them with an option to authorize themselves, or to invite someone else to (e.g. a member of their finance team). This can be done via email or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.
```

The user signing up may not have access to their business's financial data at all. For best results, provide them with an option to authorize themselves, or to invite someone else to (e.g. a member of their finance team). This can be done via email, or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.

> The user signing up may not have access to their business' financial data at all. For best results, provide them with an option to authorize themselves, or to invite someone else (e.g. a member of their finance team). This can be done via email or within your product. If the user chooses to invite someone else, this will share the Link URL with the stakeholder who has the credentials for the relevant platform.This way, users do not have to share credentials with each other, and the user with platform access can complete the authorization asynchronously. 

Reviewer: Duplicated paragraph! ;)

Author: ...how did that happen
